app/models/host/managed.rb

```diff
@@ -1023,7 +1027,7 @@ def check_if_provision_method_changed
   end
 
   def short_name_periods
-    errors.add(:name, _("must not include periods")) if (managed? && shortname && shortname.include?(".") && SETTINGS[:unattended])
+    errors.add(:name, _("must not include periods")) if (managed? && shortname && shortname.include?("."))
```

Comment:
This validation is only called when `SETTINGS[:unattended]==true`, no need to double check